Tests/js/views/services/assets/ez-locationviewviewservice-tests.js

```diff
@@ -1368,10 +1368,520 @@ YUI.add('ez-locationviewviewservice-tests', function (Y) {
         },
     });
 
+    deleteContentTest = new Y.Test.Case({
+        "name": "eZ Location View View Service delete content tests",
+
+        setUp: function () {
+            this.app = new Mock();
+            this.capi = new Mock();
+            this.content = new Mock();
+            this.location = new Mock();
+            this.languageCode = 'pol-PL';
+            this.request = {params: {languageCode: this.languageCode}};
+            this.service = new Y.eZ.LocationViewViewService({
+                app: this.app,
+                capi: this.capi,
+                content: this.content,
+                location: this.location,
+                request: this.request
+            });
+        },
+
+        tearDown: function () {
+            this.service.destroy();
+            delete this.service;
+        },
+
+        "Should fire the confirmBoxOpen event": function () {
+            var confirmBoxOpenEvent = false;
+
+            this.service.on('confirmBoxOpen', function (e) {
+                confirmBoxOpenEvent = true;
+                Assert.isObject(e.config, "The event facade should contain a config object");
+                Assert.isString(e.config.title, "The title should be defined");
+                Assert.isFunction(e.config.confirmHandler, "A confirmHandler should be provided");
+            });
+
+            this.service.fire('whatever:deleteContentAction', {content: {}});
+            Assert.isTrue(confirmBoxOpenEvent, "The confirmBoxOpen event should have been fired");
+        },
+
+        "Should notify the user when starting to delete the content item": function () {
+            var content = new Mock(),
+                parentLocation = new Mock(),
+                contentName = 'pierlugi-collina',
+                contentId = 'aContentId',
+                notified = false;
+
+            this.service.set('path', [{location: parentLocation}]);
+            this.service.set('content', content);
+
+            Mock.expect(content, {
+                method: 'get',
+                args: [Mock.Value.String],
+                run: function (attr) {
+                    if (attr === "name") {
+                        return contentName;
+                    } else if (attr === "id") {
+                        return contentId;
+                    } else {
+                        Y.fail("Unexpected parameter for content mock");
+                    }
+
+                }
+            });
+
+            Mock.expect(parentLocation, {
+                method: 'get',
+                args: ['id'],
+                returns: 'alan-shearer'
+            });
+
+            Mock.expect(this.location, {
```

Comment:
this is configuring the Location mock to have a `deleteContent` method which does not exist in the original Location Model so this method never called anywhere. If you remove that, the test still passes.
Note: the `parentLocationId` is not retrieved anywhere, the view service already has the path to the Location being displayed, so the parent Location is taken from it and that's the point of the line `this.service.set('path', [{location: parentLocation}]);` a few lines above.